<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  <title>Journals - Boden Lab</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  
  <link rel="icon" href="http://bodenlab.github.io/favicon.png">

  
  
  <link rel="stylesheet" href="/css/style.min.02c5adb90cb5a19a029b759e2533ef7be498c6ecee01e323ed0e7df6be9ebc1d.css">
  

  

</head>

<body class='page page-default-single'>
  <div id="main-menu-mobile" class="main-menu-mobile">
  <ul>
    
    
    <li class="menu-item-home">
      <a href="/">
        <span>Home</span>
      </a>
    </li>
    
    <li class="menu-item-tools">
      <a href="/tools/">
        <span>Tools</span>
      </a>
    </li>
    
    <li class="menu-item-people">
      <a href="/people/">
        <span>People</span>
      </a>
    </li>
    
    <li class="menu-item-publications">
      <a href="/publications/">
        <span>Publications</span>
      </a>
    </li>
    
  </ul>
</div>
  <div class="wrapper">
    <div class='header'>
  <div class="container">
    <div class="logo">
      <a href="http://bodenlab.github.io/"><img alt="Logo" src="/images/logo.gif" /></a>
    </div>
    <div class="logo-mobile">
      <a href="http://bodenlab.github.io/"><img alt="Logo" src="/images/logo.gif" /></a>
    </div>
    <div id="main-menu" class="main-menu">
  <ul>
    
    
    <li class="menu-item-home">
      <a href="/">
        <span>Home</span>
      </a>
    </li>
    
    <li class="menu-item-tools">
      <a href="/tools/">
        <span>Tools</span>
      </a>
    </li>
    
    <li class="menu-item-people">
      <a href="/people/">
        <span>People</span>
      </a>
    </li>
    
    <li class="menu-item-publications">
      <a href="/publications/">
        <span>Publications</span>
      </a>
    </li>
    
  </ul>
</div>
    <button id="toggle-main-menu-mobile" class="hamburger hamburger--slider" type="button">
  <span class="hamburger-box">
    <span class="hamburger-inner"></span>
  </span>
</button>
  </div>
</div>


    
    
    
    

    
    <div class="container pt-2 pt-md-6 pb-3 pb-md-6">
      <div class="row">
        <div class="col-12 col-md-3 mb-3">
          <div class="sidebar">
            
<div class="docs-menu">
  <h4>Publications</h4>
  <ul>
    
    <li class="">
      <a href="http://bodenlab.github.io/publications/preprints/">Pre-prints of note</a>
    </li>
    
    <li class="active ">
      <a href="http://bodenlab.github.io/publications/journals/">Journals</a>
    </li>
    
    <li class="">
      <a href="http://bodenlab.github.io/publications/proceedings/">Refereed conference proceedings</a>
    </li>
    
    <li class="">
      <a href="http://bodenlab.github.io/publications/book_chapters/">Book chapters</a>
    </li>
    
  </ul>
</div>

          </div>
        </div>
        <div class="col-12 col-md-9">
          
<h1 class="title">Journals</h1>
<div class="content ">
  <ul>
<li>O'Connor T, Grant CE, Bodén M and Bailey TL (2020) T-Gene: Improved target gene prediction. <em>Bioinformatics</em>. DOI <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa227/5815978?guestAccessKey=aa625a49-a2aa-4d7a-858e-8bc82867a534">10.1093/bioinformatics/btaa227</a><a href="http://biorxiv.org/cgi/content/short/803221v1">Pre-print</a></li>
<li>Fraser JA, et al. (2020) Common regulatory targets of NFIA, NFIX and NFIB during postnatal cerebellar development. <em>The Cerebellum</em>. 19:89–101. DOI <a href="https://doi.org/10.1007/s12311-019-01089-3">10.1007/s12311-019-01089-3</a></li>
<li>Horsefield S, et al. (2019) NAD+ cleavage activity by animal and plant TIR domains in cell death pathways. <em>Science</em> 365(6455):793-799. DOI <a href="https://doi.org/10.1126/science.aax1911">10.1126/science.aax1911</a> <a href="https://science.sciencemag.org/content/365/6455/793">Link direct to Science</a></li>
<li>Foley G, Sützl L, D'Cunha SA, Gillam EMJ and Bodén M (2019). SeqScrub: A web tool for automatic cleaning and annotation of FASTA file headers for bioinformatic applications. <em>Biotechniques</em>. 67(2). DOI <a href="https://doi.org/10.2144/btn-2018-0188">10.2144/btn-2018-0188</a></li>
<li>Littmann M, Goldberg T, Seitz S, Bodén M and Rost B (2019). Detailed prediction of protein sub-nuclear localization. <em>BMC Bioinformatics</em>. 20(1):205. DOI <a href="https://doi.org/10.1186/s12859-019-2790-9">10.1186/s12859-019-2790-9</a></li>
<li>Sützl L, Foley G, Gillam EM, Bodén M and Haltrich D (2019). The GMC family of oxidoreductases revisited: analysis and evolution of fungal GMC oxidoreductases. <em>Biotechnology for Biofuels</em>. 12:118. DOI <a href="https://doi.org/10.1186/s13068-019-1457-0">10.1186/s13068-019-1457-0</a></li>
<li>Fraser J, Essebier A, Brown A, Ayala Davila R, Sengar A, Tu A, Ensbey K, Day B, Scott M, Gronostajski R, Wainwright B, Bodén M, Harvey T and Piper M. (2019) Granule neuron precursor cell proliferation is regulated by NFIX and intersectin 1 during postnatal cerebellar development. <em>Brain Structure and Function</em>. 224:2, pp 811–827. DOI <a href="https://doi.org/10.1007/s00429-018-1801-3">10.1007/s00429-018-1801-3</a></li>
<li>Gumulya Y, et al. (2019) Engineering thermostable CYP2D enzymes for biocatalysis using combinatorial libraries of ancestors for directed evolution (CLADE). <em>ChemCatChem</em>. 11:2, pp 841-850. DOI <a href="https://doi.org/10.1002/cctc.201801644">10.1002/cctc.201801644</a></li>
<li>Gumulya Y, et al. (2018) Engineering highly functional thermostable proteins using ancestral sequence reconstruction. <em>Nature Catalysis</em>. 1, pp 878–888. DOI <a href="https://doi.org/10.1038/s41929-018-0159-5">10.1038/s41929-018-0159-5</a> <a href="https://rdcu.be/9Pia">Free access</a></li>
<li>Fletcher SJ, Bodén M, Mitter N and Carroll BJ (2018) SCRAM: a pipeline for fast index-free small RNA read alignment and visualization. <em>Bioinformatics</em>. 34:15, pp 2670–2672. DOI <a href="http://dx.doi.org/10.1093/bioinformatics/bty161">10.1093/bioinformatics/bty161</a><a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty161/4938488?guestAccessKey=2d418e5c-62e3-4608-af5e-b39b40c26ac3">Free access</a></li>
<li>Zaugg J, Gumulya Y, Bodén M, Mark AE and Malde AK (2018) Effect of Binding on Enantioselectivity of Epoxide Hydrolase. <em>Chem Inf Model</em>. 58 (3): 630–640. DOI <a href="https://doi.org/10.1021/acs.jcim.7b00353">10.1021/acs.jcim.7b00353</a>.</li>
<li>Zaugg J, Gumulya Y, Malde AK and Bodén M (2017) Learning Epistatic Interactions from Sequence-Activity Data to Predict Enantioselectivity. <em>J Comput Aided Mol Des</em>. 31(12):1085–1096. DOI <a href="https://doi.org/10.1007/s10822-017-0090-x">10.1007/s10822-017-0090-x</a> <a href="http://rdcu.be/BdRu">Free access</a></li>
<li>Essebier A, Lamprecht M, Piper M and Bodén M (2017) Bioinformatics approaches to predict target genes from transcription factor binding data. <em>Methods</em>. Volume 131, Pages 111-119. DOI <a href="http://dx.doi.org/10.1016/j.ymeth.2017.09.001">10.1016/j.ymeth.2017.09.001</a>.</li>
<li>Bernhofer M, Goldberg T, Wolf S, Ahmed M, Zaugg J, Bodén M and Rost B (2017) NLSdb - major update for database of nuclear localization signals and nuclear export signals, <em>Nucleic Acids Res</em>. 46:D1, pp D503–D508. DOI <a href="http://dx.doi.org/10.1093/nar/gkx1021">10.1093/nar/gkx1021</a></li>
<li>Patrick R, Kobe B, Le Cao KA and Bodén M (2017) PhosphoPICK-SNP: Quantifying the effect of amino acid variants on protein phosphorylation, <em>Bioinformatics</em>. 33 (12): 1773-1781. DOI <a href="http://dx.doi.org/10.1093/bioinformatics/btx072">10.1093/bioinformatics/btx072</a>. <a href="https://academic.oup.com/bioinformatics/article/33/12/1773/2982049/PhosphoPICKSNP-quantifying-the-effect-of-amino?guestAccessKey=eec63034-0aec-4fa6-b13a-7cbbb167c98d">Free access</a></li>
<li>O'Connor TR, Bodén M and Bailey, TL (2017) CISMAPPER: predicting regulatory interactions from transcription factor ChIP-seq data. <em>Nucleic Acids Res</em> 45 (4): e19. DOI: <a href="http://dx.doi.org/10.1093/nar/gkw956">10.1093/nar/gkw956</a>.</li>
<li>Williams SJ, Yin L, Foley G, Casey LW, Outram MA, Ericsson DJ, Lu J, Bodén M, Dry I and Kobe B (2016) Structure and function of the TIR domain from the grape NLR protein RPV1. <em>Frontiers in Plant Science</em>. DOI: <a href="http://dx.doi.org/10.3389/fpls.2016.01850">10.3389/fpls.2016.01850</a></li>
<li>Fraser J, Essebier A, Gronostajski RM, Bodén M, Wainwright BJ, Harvey TJ and Piper M (2016) Cell type-specific expression of NFIX in the developing and adult cerebellum. <em>Brain Structure and Function</em>. 222:5, pp 2251–2270. DOI: <a href="http://dx.doi.org/10.1007/s00429-016-1340-8">10.1007/s00429-016-1340-8</a></li>
<li>Harris L, Zalucki O, Gobius I, Osinski J, Harvey T, Essebier A, Vidovic D, GladwynNg I, McDonald H, Burne T, Heng J, Richards L, Gronostajski R and Piper M (2016) Transcriptional regulation of intermediate progenitor cell generation during hippocampal development. <em>Development</em>. 143: 4620-4630; DOI: <a href="http://dx.doi.org/10.1242/dev.140681">10.1242/dev.140681</a></li>
<li>Patrick R, Horin C, Kobe B, Le Cao KA and Bodén M (2016) Prediction of kinase-specific phosphorylation sites through an integrative model of protein context and sequence. BBA - <em>Proteins and Proteomics</em>. 1864(11):1599-608. DOI:<a href="http://dx.doi.org/10.1016/j.bbapap.2016.08.001">10.1016/j.bbapap.2016.08.001</a>.</li>
<li>Essebier A, Vera Wolf P, Cao MD, Carroll BJ, Balasubramanian S and Bodén M (2016) Statistical enrichment of epigenetic states around triplet repeats that undergo expansions. <em>Frontiers in Neuroscience</em>. 10:92. DOI: <a href="http://dx.doi.org/10.3389/fnins.2016.00092">10.3389/fnins.2016.00092</a> (Open access)</li>
<li>Cao MD, Allison L, Dix TI and Bodén M (2016) Robust Estimation of Evolutionary Distances with Information Theory. <em>Molecular Biology and Evolution</em>. 33 (5): 1349-1357. DOI: <a href="http://dx.doi.org/10.1093/molbev/msw019">10.1093/molbev/msw019</a></li>
<li>Huang W, Johnston W, Bodén M and Gillam EM (2016) ReX - A suite of computational tools for the design, visualization and analysis of chimeric protein libraries. <em>BioTechniques</em>. Vol. 60, No. 2, pp. 91–94. DOI: <a href="http://dx.doi.org/10.2144/000114381">10.2144/000114381</a>.</li>
<li>Oyarzun P, Ellis JJ, Gonzalez-Galarza FF, Jones AR, Middleton D, Bodén M and Kobe B (2015) A bioinformatics tool for epitope-based vaccine design that accounts for human ethnic diversity: Application to emerging infectious diseases. <em>Vaccine</em>. 33(10):1267-73. DOI: <a href="http://dx.doi.org/10.1016/j.vaccine.2015.01.040">10.1016/j.vaccine.2015.01.040</a></li>
<li>Patrick R, Le Cao KA, Kobe B and Bodén M (2015) PhosphoPICK: Modelling cellular context to map kinase-substrate phosphorylation events. <em>Bioinformatics</em>. 31(3):382-389. DOI: <a href="http://dx.doi.org/10.1093/bioinformatics/btu663">10.1093/bioinformatics/btu663</a></li>
<li>Cao MD, Balasubramanian S and Bodén M (2015) Sequencing technologies and tools for short tandem repeat variation detection, <em>Briefings in Bioinformatics</em>. 16 (2): 193-204. DOI:<a href="http://dx.doi.org/10.1093/bib/bbu001">10.1093/bib/bbu001</a>.</li>
<li>Rona G, Borsos M, Ellis J, Mehdi A, Christie M, Kornyei Z, Neubrandt M, Toth J, Bozoky Z, Buday L, Madarasz E, Bodén M, Kobe B and Vertessy B (2014). Dynamics of re-constitution of the human nuclear proteome after cell division is regulated by NLS-adjacent phosphorylation. <em>Cell Cycle</em>. 13(22). DOI: <a href="http://dx.doi.org/10.4161/15384101.2014.960740">10.4161/15384101.2014.960740</a> (author eprint)</li>
<li>Mehdi AM, Patrick R, Bailey TL and Bodén M (2014) Predicting the dynamics of protein abundance, <em>Molecular &amp; Cellular Proteomics</em>. May;13(5):1330-40. DOI:<a href="http://dx.doi.org/10.1074/mcp.M113.033076">10.1074/mcp.M113.033076</a>.</li>
<li>Cao MD, Tasker E, Willadsen K, Imelfort M, Vishwanathan S, Sureshkumar S, Balasubramanian S and Bodén M (2014) <em>Inferring Short Tandem Repeat Variation from Paired-End Short Reads, Nucleic Acids Research</em>. Feb;42(3):e16. DOI: <a href="http://dx.doi.org/10.1093/nar/gkt1313">10.1093/nar/gkt1313</a>.</li>
<li>Mazgut J, Tino P, Bodén M and Yan H (2014) <em>Dimensionality Reduction and Topographic Mapping of Binary Tensors, Pattern Analysis and Applications</em>. 17(3):497-515. DOI: <a href="http://dx.doi.org/10.1007/s10044-013-0317-y">10.1007/s10044-013-0317-y</a>.</li>
<li>Chang C-W, Counago RLM, Williams S, Bodén M and Kobe B (2013) The distribution of different classes of nuclear localization signals (NLSs) in different organisms and the utilization of the minor NLS-binding site in plant nuclear import factor importin-alpha, <em>Plant Signaling &amp; Behavior</em>, 8(10). DOI: <a href="http://dx.doi.org/10.4161/psb.25976">10.4161/psb.25976</a>.</li>
<li>Chang C-W, Counago RLM, Williams S, Bodén M and Kobe B (2013) <em>Distinctive Conformation of Minor Site-Specific Nuclear Localization Signals Bound to Importin-alpha, Traffic</em>. Nov;14(11):1144-54. DOI: <a href="http://dx.doi.org/10.1111/tra.12098">10.1111/tra.12098</a></li>
<li>Oyarzun P, Ellis JJ, Bodén M and Kobe B (2013) PREDIVAC: <em>CD4+ T-cell epitope prediction for vaccine design that covers 95% of human HLA class II DR protein diversity, BMC Bioinformatics</em>, 14:52. DOI: <a href="http://dx.doi.org/10.1186/1471-2105-14-52">10.1186/1471-2105-14-52</a></li>
<li>Willadsen K, Cao MD, Wiles J, Balasubramanian SK and Bodén M (2013) <em>Repeat-encoded poly-Q tracts show statistical commonalities across species, BMC Genomics</em> 14:76. DOI: <a href="http://dx.doi.org/10.1186/1471-2164-14-76">10.1186/1471-2164-14-76</a></li>
<li>Mehdi A, Sehgal S, Kobe B, Bailey TL and Bodén M (2013) DLocalMotif: <em>A discriminative approach for discovering local motifs in protein sequences, Bioinformatics</em>, 29(1):39-46. DOI: <a href="http://dx.doi.org/10.1093/bioinformatics/bts654">10.1093/bioinformatics/bts654</a>.</li>
<li>Chang C-W, Counago RLM, Williams S, Bodén M and Kobe B (2012) <em>Crystal Structure of Rice Importin-alpha and Structural Basis of its Interaction with Plant-Specific Nuclear Localization Signals, The Plant Cell</em>, 24(12):5074-88. doi: <a href="http://dx.doi.org/10.1105/tpc.112.104422">10.1105/tpc.112.104422</a>. (Access)</li>
<li>Patrick R, Le Cao K-A, Davis M, Kobe B and Bodén M (2012) Mapping the stabilome: a novel computational method for classifying metabolic protein stability. <em>BMC Systems Biol</em> 6(1):60. (Open access.)</li>
<li>Kobe B &amp; Bodén M (2012) Computational modelling of linear motif-mediated protein interactions. <em>Curr Top Med Chem</em>. 12 (14), 1553-1561 [PMID: 22827524]</li>
<li>Willadsen K, Mohamad N, and Bodén M (2012) NSort/DB: An intra-nuclear compartment protein database. <em>Genomics, Proteomics &amp; Bioinformatics</em> 10(4):226-229. DOI: <a href="http://dx.doi.org/10.1016/j.gpb.2012.07.001">10.1016/j.gpb.2012.07.001</a></li>
<li>Madala PK, Fairlie DP and Bodén M (2012) Matching cavities in G protein-coupled receptors to infer ligand-binding sites. <em>Journal of Chemical Information &amp; Modeling</em>. 52(5):1401-10.</li>
<li>Bauer DC, Willadsen K, Buske FA, Le Cao K-A, Bailey TL, Dellaire G and Bodén M (2011) Sorting the nuclear proteome. <em>Bioinformatics</em>. 27(13):i7-14.</li>
<li>Mehdi A, Sehgal S, Kobe B, Bailey TL and Bodén M (2011) A probabilistic model of nuclear import of proteins. <em>Bioinformatics</em>. 27(9):1239-46.</li>
<li>Marfori M, Mynott A, Ellis JJ, Mehdi AM, Saunders NF, Curmi PM, Forwood JK, Bodén M and Kobe B (2011) Molecular basis for specificity of nuclear import and prediction of nuclear localization. <em>Biochim Biophys Acta</em> 1813(9):1562-77.</li>
<li>You L, Brusic VL, Gallagher M and Bodén M (2010) Using Gaussian process with test rejection to detect T-cell epitopes in pathogen genomes. <em>IEEE/ACM Trans Comp Biol Bioinformatics</em> 7(4):741-751.</li>
<li>Mohamad N and Bodén M (2010) The proteins of intra-nuclear bodies: a data-driven analysis of sequence, interaction and expression. <em>BMC Systems Biology</em> 4:44.</li>
<li>Buske FA, Bodén M, Bauer DC and Bailey TL (2010) Assigning roles to DNA regulatory motifs using comparative genomics. <em>Bioinformatics</em> 26:860-6.</li>
<li>Bauer DC, Buske FA, Bailey TL and Bodén M (2010) Predicting SUMOylation sites in developmental transcription factors of Drosophila melanogaster. <em>Neurocomputing</em>. 73(13-15): 2300-2307.</li>
<li>Bodén M, Dellaire G, Burrage K and Bailey TL (2010) <em>A Bayesian network model of proteins&rsquo; association with Promyelocytic leukemia (PML) nuclear bodies</em>. J Comp Biol 17(4): 617-30.</li>
<li>Bailey T, Bodén M, Whitington T and Machanick P (2010) The value of position-specific priors in motif discovery using MEME. <em>BMC Bioinformatics</em> 11:179.</li>
<li>Buske FA, Maetschke S and Bodén M (2009) It's about time: signal recognition in staged models of protein translocation. <em>Pattern Recognit</em> 42(4): 567-574.</li>
<li>Buske FA, Thier R, Gillam EM and Bodén M (2009) In silico characterisation of protein chimeras: relating sequence and function within the same fold. <em>Proteins</em> 77(1), pp. 111-120.</li>
<li>Bailey TL, Bodén M, Buske FA, Frith M, Grant CE, Clementi L, Ren J, Li WW and Noble WS (2009) MEME Suite: tools for motif discovery and searching. <em>Nucleic Acids Res</em> 37:W202-8.</li>
<li>Bodén M and Teasdale RD (2008) <em>Determining nucleolar association from sequence by leveraging protein-protein interactions</em>. J Comp Biol 15(3): 291-304.</li>
<li>Bodén M and Bailey TL (2008) Associating transcription factor binding site motifs with target GO terms and target genes. <em>Nucleic Acids Res</em> 36(12): 4108-4117.</li>
<li>Bodén M and Bodén M (2007) Evolving spelling exercises to suit individual student needs. <em>Appl Soft Comput</em> 7: 126-135.</li>
<li>Suksawatchon J, Lursinsap C and Bodén M (2007) <em>Computing the reversal distance between genomes in the presence of multi-gene families via binary integer programming</em>. J Bioinformatics Comp Biol 5(1): 117-133.</li>
<li>Hawkins J, Mahony D, Maetschke S, Wakabayashi M, Teasdale RD and Bodén M (2007) Identifying novel peroxisomal proteins. <em>Proteins</em>69(3): 606-616.</li>
<li>Hawkins J, Davis L and Bodén M (2007) <em>Predicting nuclear localization</em>. J Proteome Res 6(4): 1402-1409.</li>
<li>Bauer, D., Bodén M., Thier, R. and Gillam, E. M. STAR: Predicting recombination sites from amino acid sequence. <em>BMC Bioinformatics</em>, 7:437, 2006.</li>
<li>Bodén, M. and Bailey, T. L. Identifying sequence regions undergoing conformational change via predicted continuum secondary structure. <em>Bioinformatics</em>. 22(15): 1809-1814, 2006.</li>
<li>Bodén, M., Yuan, Z. and Bailey, T. L. Prediction of protein continuum secondary structure with probabilistic models based on NMR solved structures. <em>BMC Bioinformatics</em>. 7:68, 2006.</li>
<li>Yuan, Z., Zhang, F., Davis, M. J., Bodén M. and Teasdale, R. D. Predicting the solvent accessibility of transmembrane residues from protein sequence. <em>Journal of Proteome Research</em>. 5(5):1063-1070, 2006.</li>
<li>Hawkins, J. and Bodén M. Detecting and sorting targeting peptides with recurrent networks and support vector machines. <em>Journal of Bioinformatics and Computational Biology</em>, 4(1), 2006.</li>
<li>Bodén, M. and Hawkins, J. Prediction of subcellular localisation using sequence-biased recurrent networks. <em>Bioinformatics</em>. 21(10):2279-2286, 2005.</li>
<li>Hawkins, J. and Bodén M. The applicability of recurrent neural networks for biological sequence analysis. <em>IEEE/ACM Transactions on Computational Biology and Bioinformatics</em>, 2(3): 243-253, 2005.</li>
<li>Bodén, M. and Hawkins, J. Improved access to sequential motifs: A note on the architectural bias of recurrent networks. <em>IEEE Transactions on Neural Networks</em>. 16(2), 2005.</li>
<li>Bodén, M. <em>Generalization by symbolic abstraction in cascaded recurrent networks, Neurocomputing</em>, 57, pp. 87-104, 2004.</li>
<li>Bodén, M. and Blair, A. Learning the dynamics of embedded clauses, Applied Intelligence: <em>Special issue on natural language and machine learning</em>, 19(1/2), pp. 51-63, 2003.</li>
<li>Bodén, M. and Wiles, J. On learning context free and context sensitive languages, <em>IEEE Transactions on Neural Networks</em>. 13(2), pp. 491-493, 2002.</li>
<li>Bodén, M. and Wiles, J., Context-free and context-sensitive dynamics in recurrent neural networks, <em>Connection Science</em>, 12 (3/4), pp. 197-210, 2000.</li>
<li>Bodén, M. and Niklasson, L., Semantic systematicity and context in connectionist Networks, <em>Connection Science</em>, 12 (2), pp. 111-142, 2000.</li>
</ul>

</div>
</div>

        </div>
      </div>
    </div>
    
  </div>

  <div class="sub-footer">
  <div class="container">
    <div class="row">
      <div class="col-12">
        <div class="sub-footer-inner">
          <ul>
            <li class="zerostatic"><a href="https://www.zerostatic.io">www.zerostatic.io</a></li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>

  

  
  

  
  <script type="text/javascript" src="/js/scripts.min.1237ff71925bb8625c97a9af8db4c54525258bedfd7c47493daaff723bea755e.js"></script>
  

  
  
  
    
  


</body>

</html>
